libs/model/src/globalActivityCache.ts

```diff
@@ -6,15 +6,21 @@ import { DB } from './models/index';
 
 export async function getActivityFeed(models: DB, id = 0) {
   /**
-   * Last 50 updated threads
+   * Last 50 updated threads and their comments
    */
-
-  const filterByCommunityForUsers = id
-    ? 'A.community_id = T.community_id and'
-    : '';
-
   const query = `
-    WITH ranked_thread_notifications AS (
+    ${
```

Comment:
I think the tertiary is unnecessary here, since the planner will just choose not to perform this CTE if it is not used in the body of the sql (I might be wrong here).

Reply:
Good catch - analyzed query plan and it indeed is not used if the other ternary resolves to an empty string;